nodes/core/core/89-delay.js

```diff
@@ -105,15 +105,15 @@ module.exports = function(RED) {
         }
         else if (node.pauseType === "delayv") {
             node.on("input", function(msg) {
-                var delayvar = Number(msg.delay || 0);
-                if (delayvar < 0) { delayvar = 0; }
+                var delayvar = Number(msg.delay || node.timeout);
```

Comment:
If `msg.delay` is set, but is `0`, then `node.timeout` will be applied. This means it isn't possible for a flow to elect to not delay a given message (by setting `msg.delay` to 0).

Reply:
true - will make it a proper test of 0 rather than falsey